apps/webapp/app/components/run/TaskDetail.tsx

```diff
@@ -32,7 +32,7 @@ import { ClientOnly } from "remix-utils";
 import { Spinner } from "../primitives/Spinner";
 import type { DetailedTask } from "~/routes/_app.orgs.$organizationSlug.projects.$projectParam.jobs.$jobParam.runs.$runParam.tasks.$taskParam/route";
 
-export function TaskDetail({ task }: { task: DetailedTask }) {
+export function TaskDetail({ task }: { task: DetailedTask & { params: {} } }) {
```

Comment:
I don't understand why this change is needed

Reply:
It was giving a type error like params property does not exist so had to manually add it.